files/mygui/openmw_resources.xml

```diff
@@ -114,6 +114,24 @@
         </Widget>
     </Resource>
 
+    <Resource type="ResourceLayout" name="MW_ComboBoxNoTransp" version="3.2.0">
```

Comment:
As far as I recall, ComboBox is only used once so you can just change the original skin instead.